charts/loki-distributed/templates/ingress.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
should we also check to make sure the gateway is disabled here? I'd like to enforce the 1 or the other restriction in code if possible.

Reply:
Sounds like a good idea.  I'll take a look at using the validate template for this today, I haven't used one for that yet.